README.md

```diff
@@ -329,17 +329,23 @@ cd build
 cmake -GNinja ../ -DSYCL_COMPILER=dpcpp
 ninja
 ```
+The target triplet can be set by adding `-DDPCPP_SYCL_TARGET=<triplet>`. If it
+is not set, the default values is `spir64_x86_64-unknown-unknown`, which targets
+the CPU. Other possible triplets targeting GPUs are: `spir64-unknown-unknown`
+(Intel), `nvptx64-nvidia-cuda` (Nvidia), and `amdgcn-amd-amdhsa` (AMD).
 
-If building for `AMD_GPU` devices, it is also required to provide the
-specific architecture through the `DPCPP_SYCL_ARCH` CMake variable.
+If building for NVIDIA or AMD devices, it is advisable to specify the actual
```

Comment:
We can call it as advisable for nvidia but required for AMD.

Reply:
I rephrased the sentence.